website/newsletter.py

```diff
@@ -48,14 +48,31 @@ def run():
     threading.Thread(target=run, daemon=True).start()
 
 
+def get_subscription_status(email):
+    """ Fetches the subscription status of the user. Values can be 'subscribed',
+    'unsubscribed' or '' if the user has never subscribed """
+    if email and MAILCHIMP_API_URL:
+        response = get_mailchimp_subscriber(email)
+        if response and response.status_code == 200:
+            return response.json().get('status')
+    return ''
+
+
+def create_subscription(email, country, teacher=True, created_class=False, customized_class=False):
```

Comment:
Do you think in the future we might add more tags to this functions or these two will be enough? I mention it, because this function might quickly have tons of parameters, it has happened to us in the past.